src/argus/htmx/middleware.py

```diff
@@ -61,6 +66,14 @@ class HtmxMessageMiddleware(MiddlewareMixin):
 
     TEMPLATE = "messages/_notification_messages_htmx_append.html"
 
+    def process_exception(self, request, exception):
+        error_msg = "500 Internal Server Error"
+        messages.error(request, error_msg)
+        if str(exception):
+            error_msg += f"{error_msg}: {exception}"
+        LOG.error(error_msg)
```

Comment:
```suggestion
        LOG.exception(error_msg)
```

to also print traceback might be nice

Reply:
I'm not sure the traceback is always available at this point but why not.